storage-node-v10-ha-ra-grs-v4/index.js

```diff
@@ -0,0 +1,135 @@
+// Configure .env file with appropriate environmental variables
+// for your storage account.
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').load();
+}
+
+const {
+  BlobServiceClient,
+  SharedKeyCredential,
+  uploadFile,
+  newPipeline
+} = require('@azure/storage-blob');
+
+// Set up user input.
+const readline = require('readline');
+const readInput = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+// Read environment variables for storage access.
+const accountName = process.env.AZURE_STORAGE_ACCOUNT_NAME;
+const storageAccessKey = process.env.AZURE_STORAGE_ACCOUNT_ACCESS_KEY;
+const sharedKeyCredential = new SharedKeyCredential(accountName, storageAccessKey);
+const primaryAccountURL = `https://${accountName}.blob.core.windows.net`;
+const secondaryAccountURL = `https://${accountName}-secondary.blob.core.windows.net`;
```

Comment:
Is -secondary a standard naming convention or is this something that the sample requires.  If the user has to do anything to set this up, then you need to let them know that they need to name it this way.